content/installation/node/YAML-properties-node.md

```diff
@@ -0,0 +1,162 @@
+<!--
+title: "YAML configuration properties for the Node agent "
+description: "Instructions for configuring Node agent properties via YAML file"
+tags: "installation node agent YAML configuration rules properties"
+-->
+
+Contrast support YAML-based configuration for the Node agent. This allows you to store configuration on disk with environment variable and CLI args. overrides.
+
```

Comment:
Is there a better way to say "environment variable and CLI args. overrides."?

Reply:
How about "This allows you to store configuration on disk that you can override with environment variables or command line arguments."

Also, should that be "Contrast supports"?